client/state/global-sidebar/selectors.ts

```diff
@@ -63,14 +63,18 @@ export const getShouldShowCollapsedGlobalSidebar = (
 	const pluginsScheduledUpdatesEditMode =
 		state.route.path?.current?.includes( 'scheduled-updates/edit' ) ||
 		state.route.path?.current?.includes( 'scheduled-updates/create' );
+	const isBulkDomainsDashboard = state.route.path?.current === '/domains/manage';
```

Comment:
Same note as before https://github.com/Automattic/wp-calypso/pull/90785#discussion_r1603436188

I just updated this from using `.endsWith` to use a strict equality check. It shouldn't make any noticeable changes here but seems more robust (ex - maybe in the future there is a separate .com/siteName/domains/manage route and .endsWith would allow bleed over).